dashboard/app/models/artifact.rb

```diff
@@ -0,0 +1,33 @@
+# == Schema Information
+#
+# Table name: artifacts
```

Comment:
Naming nit:  I don't love the term "artifact";  my courses never had "artifacts" (except for that one time I signed up for Professor Jones class, which was a big mistake, let me tell you.)  

Is there any more familiar term we can use?  ("task", "work", "doc", I don't know what.)